src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,55 @@
+package main.java.tasklist;
+
+import main.java.tasks.Task;
+import java.util.ArrayList;
+
+public class TaskList {
+    protected ArrayList<Task> tasks;
+
+    public TaskList() {
+        this(new ArrayList<Task>());
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+
+    public ArrayList<Task> getTasks() {
+        return tasks;
+    }
+
+    /**
+     * Get the task with at the specific index in the array
+     * @return a Task
+     */
+    public Task getTask(int index){
+         return getTasks().get(index);
+    }
+
+    /**
+     * Add a Task to the TaskList
+     * @param task
+     */
+    public void addTask(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Delete the task with at the specific index in the array
+     * @param index
+     */
+    public void deleteTask(int index) {
+        tasks.remove(index);
+    }
+
+    /**
+     * Mark the task with at the specific index in the array as complete
```

Comment:
I noticed a similar issue in the leading word of each Javadoc comment in other places as well.